### ISSUE TITLE
Accommodate "null" values for writing to Hbase

### ISSUE THREAD

**Issue by AbdullahServantOfAllah** [hortonworks-spark/shc#228]
I have a column say "col1" in a hbase table. In my spark Dataframe I have values only for few rows for that column(col1), for most rows the value is null. Is it possible to send null values using the SHC connector? or is there a workaround? please advice.

Thanks you for the previous answers,
and Best Regards,

**Comment by weiqingy** [comment]
Yes. SHC master branch has supported the cases with "null" values. @btomala submitted a [PR](https://github.com/hortonworks-spark/shc/pull/198) for this which was merged into SHC master branch.

**Comment by meirs84** [comment]
Hi @weiqingy any idea what is the minimal version of spark the connector should have the fix in ?

we r currently working with spark 2.6.4 and we don't seem to find a jar with the fix
